orchestration-stub/src/services/request-parameters.ts

```diff
@@ -48,3 +51,15 @@ const getReauthenticate = (form: ParsedUrlQuery): string | undefined => {
     return form.reauthenticate;
   }
 };
+
+const getChannel = (channel: string | string[] | undefined): ChannelEnum => {
+  if (
+    typeof channel === "string" &&
+    (channel === ChannelEnum.NONE ||
+      channel === ChannelEnum.WEB ||
+      channel === ChannelEnum.STRATEGIC_APP)
```

Comment:
Overkill really, this is just something I thought of but no requirement to change this.

You may be able to "genericise" this logic to a type predicate: https://www.typescriptlang.org/docs/handbook/2/narrowing.html#using-type-predicates

Something like:
```
const isChannel(channel: any): channel is ChannelEnum => typeof channel === 'string' && Object.values(ChannelEnum).includes(channel);

const getChannel = (channel: string | string[] | undefined): ChannelEnum => {
  if (isChannel(channel)) {
    return channel;
  }
  throw new Error("Unknown channel: " + channel);
};
```